### ISSUE TITLE
Data tracking not recording active memory in aggregation configuration

### ISSUE THREAD

**Issue by DavidPoliakoff** [LLNL/Caliper#190]
I have two configurations 
```
  if(memoryEvents){
    caliper_kokkos_track_memory = true;
    additional_services+=":alloc";
  }
  cali::config_map_t default_config {
   {"CALI_RECORDER_FILENAME",fileOutput},
   {"CALI_ALLOC_RECORD_ACTIVE_MEM","true"},
   {"CALI_SERVICES_ENABLE","timestamp:event:aggregate:recorder" + additional_services}
  };
  cali::config_map_t trace_config {
   {"CALI_RECORDER_FILENAME",fileOutput},
   {"CALI_SERVICES_ENABLE","timestamp:event:trace:recorder" + additional_services}
  };
```

When I run in the trace mode, I see memory events and allocation sizes, everything works, but in aggregation mode I'm not seeing the allocation sizes. Any thoughts?

**Comment by daboehme** [comment]
Aggregation class meta-info was missing from `alloc.total_size` attribute. Should be fixed with 34ccf2b0b2cfef25286cf0d125af0df1b73f5fef, please check.

**Comment by DavidPoliakoff** [comment]
This is fixed, closing. Thanks again!
